PreProcessing/generate_dummy_motion.m

```diff
@@ -1,7 +1,12 @@
 function [Qs] = generate_dummy_motion(S,model_info,n_data_points)
 % --------------------------------------------------------------------------
 % generate_dummy_motion
-%   Generate a set of coordinate values for an OpenSim model.
+%   Generate a set of coordinate values for an OpenSim model. Coordinate
+%   values are taken in the range of motion. Upper and lower bounds are
+%   determined from (in ascending order of priority)
```

Comment:
should it not be descending order of priority? I.e. those coming from S.misc.msk_geom_bounds have the highest priority.